gen/definitions/wireless_ssid.yaml

```diff
@@ -0,0 +1,524 @@
+name: Wireless SSID
+rest_endpoint: /networks/%v/wireless/ssids
```

Comment:
Can't we change this to `/networks/%v/wireless/ssids/%v`, which should then also fix the `getPath()` function?

Reply:
changed as suggested

Reply:
I think in combination with "put_create" it should work.

Reply:
it will still require manual changes inside read and update functions because the generation logic is as follows:
```
        {{- if .PutCreate}}
	res, err := r.client.Put(plan.getPath(), body)
	{{- else}}
	res, err := r.client.Put(plan.getPath() + "/" + url.QueryEscape(plan.Id.ValueString()), body)
	{{- end}}
```
so create would work automatically, but we would have to manually claim read and update, so in my opinion it is better to only customize create